doc: add installation instructions to README

guisp03 opened the pull request:
This pull request addresses issue #77 by adding a new section to the README.md with installation instructions for the game. The instructions include:
- Prerequisites, such as Python 3 and the Pygame engine.
- Step-by-step guidance for installing Pygame, covering both pip and alternative package managers (e.g., pacman for Arch-based systems).
- Instructions on how to clone the repository, navigate to the project folder, and run the game using the coral.py file.

These additions aim to provide a clear and accessible guide for new developers or users unfamiliar with Python and Pygame, ensuring an easier setup process.

ImLusca commented on README.md:
```diff
@@ -40,4 +40,51 @@
+
+ ### Prerequisites
+ 
+ - Python 3: Make sure you have Python 3 installed on your machine. You can download it from the official Python website.
```
Add python download link using `download it from the official Python [website](https://www.python.org/downloads/)`

ImLusca replied:
Add the apt install command for linux users

ImLusca commented on README.md:
````diff
@@ -40,4 +40,51 @@
+ First, clone the Coral repository to your local machine by running the following command in your terminal:
+
+ ```bash
+ git clone https://github.com/your-username/coral.git
````
github http auth is deprecated. Use SSH instead

guisp03 commented:
Hi @ImLusca , I’ve made the requested changes:
- Added the Python download link for Windows/Mac users.
- Included the `apt install` command for Linux users with a note on distribution differences.
- Updated the clone command to SSH.
Let me know if there’s anything else to adjust. Thanks for the feedback!

samuks123 reviewed (approved):
All the requested changes have been resolved
